les/client.go

```diff
@@ -99,6 +99,7 @@ func New(stack *node.Node, config *ethconfig.Config) (*LightEthereum, error) {
 			chainConfig: chainConfig,
 			iConfig:     light.DefaultClientIndexerConfig,
 			chainDb:     chainDb,
+			lespayDb:    lespayDb,
```

Comment:
I'm not sure `lespay` is a good name acutally. Now  in the client side we store lots of server statistics and in the server side we store some client balances.
Maybe `lesdb` is better.